plugin.video.wizjatv/wizja.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+
+import sys
+reload(sys)
```

Comment:
To nie wiem po co jest, ale na razie nie sugeruję zmiany, bo jeszcze coś popsuję :-)
Kiedyś się wyczyści takie kwiatki.